manual/Creating_a_table/Creating_a_sharded_table/Table_sharding_current_limitations.md

```diff
@@ -0,0 +1,14 @@
+# Manual Table Sharding: Current Limitations
+
+When creating a sharded table, you should be aware of the current limitations you will face while working with it.
+
+1. You are not allowed to mix local table creation on the first server in the cluster and a clustered table on the second cluster. When you create a sharded table, you should try to use the same node you used for creation to interact with and send all queries to it without mixing different nodes. This limitation is due to an issue when we initialize the internal map of the shards once you create a clustered table because of how replication works, it will rewrite all data.
```

Comment:
The first 2 sentences are very hard to comprehend. This requires rewriting and simplification.

Reply:
Now it says:
> When working with a sharded table, use the same node for all operations

This means that after I run `create table c:t ... shards='5' rf='2'`, I can only insert data into it from the same node where I created it. If that's true, we need to fix it. If it's not, we should update the docs to make it clear.